tests/other_test.go

```diff
@@ -0,0 +1,57 @@
+// Tests in this file are NOT run in the PR pipeline. They are run in the continuous testing pipeline along with the ones in pr_test.go
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+)
+
+const completeDir = "examples/complete"
+const existingICRNamespaceName = "geretain-sao-ns-do-not-delete"
+
+func TestRunCompleteExample(t *testing.T) {
+	t.Parallel()
+	const prefix = "complete-icr"
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  completeDir,
+		Prefix:        prefix,
+		ResourceGroup: resourceGroup,
+		TerraformVars: map[string]interface{}{
+			"resource_group":   resourceGroup,
+			"namespace_region": "us-south",
+			"retain_untagged":  true,
+			"namespace_name":   fmt.Sprintf("%s-ns", prefix),
+		},
+	})
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunExistingNamespaceExample(t *testing.T) {
```

Comment:
Can we move this to PR tests to ensure we test existing namespace use case as part of PR tests? Also I think we should do this test in the DA solution, and not the complete example.